include/aos/iam/certmodules/certmodule.hpp

```diff
@@ -234,7 +234,7 @@ class CertModule {
     static constexpr auto cOidExtKeyUsageServerAuth     = "1.3.6.1.5.5.7.3.1";
     static constexpr auto cOidExtKeyUsageClientAuth     = "1.3.6.1.5.5.7.3.2";
 
-    using ModuleCertificates    = StaticArray<CertInfo, cCertsPerModule>;
+    using ModuleCertificates    = StaticArray<CertInfo, cCertsPerModule + 1>;
```

Comment:
In general I'm fine with the solution if storage in zephyr doesn't rely on CERT_PER_MODULE conf value.

Reply:
Why this +1 ?

Reply:
Refactored to not extend size